lib/community/widgets/post_card_metadata.dart

```diff
@@ -353,6 +353,78 @@ class UrlPostCardMetaData extends StatelessWidget {
   }
 }
 
+class CrossPostMetaData extends StatelessWidget {
+  final int unreadComments;
```

Comment:
I would add some additional comments to each parameter for conventions

Reply:
Done